### ISSUE TITLE
关于人民币二分类正确Label判断的问题

### ISSUE THREAD

**Issue by yeahQing** [JansonYuan/Pytorch-Camp#12]
在人民币二分类任务中，训练模型时，有这两行代码
```python
# train
correct += (predicted == labels).squeeze().sum().numpy()
# valid
correct_val += (predicted == labels).squeeze().sum().numpy()
```
我不太理解(predicted == labels)这个之后为什么需要加一个squeeze()。
我在python console进行了实验，验证不加squeeze()直接进行sum()和numpy()也可以得到同样的结果
```python
# 预测的mini_batch中的结果
predicted = torch.tensor([0, 0, 1, 1, 0, 1, 1, 1, 0, 0, 0, 1, 1, 1, 0, 1])
# ground truth
labels = torch.tensor([0, 0, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 1, 1, 0, 0])
# 不加squeeze(), 不去掉为1的维度
(predicted == labels).sum().numpy()
Out[33]: array(14, dtype=int64)
# 加 squeeze(), 去掉为1的维度
(predicted == labels).squeeze().sum().numpy()
Out[34]: array(14, dtype=int64)
```
所以，老师，这个squeeze()的作用是什么呢?

**Comment by TingsongYu** [comment]
squeeze函数的功能是去掉为1的维度；这段代码中squeeze是为了适配其它情形的代码，如果predicted是2D的，就需要用了
